app/controllers/notes_controller.rb

```diff
@@ -19,7 +19,8 @@ def create
     if @note.valid?
       @note.save
 
-      redirect_to project_notes_path(@project), notice: I18n.t("note.create.success")
+      redirect_path = @note.task_level_note? ? project_task_path(@project, @note.task.id) : project_notes_path(@project)
```

Comment:
I wonder if this should redirect to a page fragment (ie `/task/123abc#task-def456`) so people aren't stuck with scrolling? Perhaps could also make use of [`:target` CSS](http://jsfiddle.net/davidThomas/NGyVL/) to highlight, but that's a bit of a refinement which may need testing.

Reply:
This is a great idea, but I wonder if it is a future improvement when we know a little more about the volume of notes that get added to tasks?